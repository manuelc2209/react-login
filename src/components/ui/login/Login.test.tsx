import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import { Login } from '.';

it('renders snapshot correctly', () => {
    const tree = renderer.create(<Login />).toJSON();
    expect(tree).toMatchSnapshot();
});

it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Login />, div);
    ReactDOM.unmountComponentAtNode(div);
});
