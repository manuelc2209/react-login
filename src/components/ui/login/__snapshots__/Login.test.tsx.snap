// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders snapshot correctly 1`] = `
<div
  className="sc-eCImPb"
>
  <div
    className="sc-iCfMLu igChiH"
  >
    <span
      className="sc-jRQBWg"
    />
    <div
      className="sc-hKwDye lgrjjs"
    >
      <input
        className="sc-gsDKAQ dUUKnI"
        onBlur={[Function]}
        onChange={[Function]}
      />
    </div>
  </div>
  <div
    className="sc-hKwDye lgrjjs"
  >
    <input
      className="sc-gsDKAQ dUUKnI"
      onBlur={[Function]}
      onChange={[Function]}
      type="password"
    />
  </div>
  <button
    className="sc-bdvvtL kKijdc sc-gKclnd"
    onClick={[Function]}
    onMouseDown={[Function]}
    onMouseUp={[Function]}
    size="small"
  >
    Submit
  </button>
</div>
`;
